.github/workflows/rust.yml

```diff
@@ -82,7 +82,16 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: -v
+          args: -v  
+      - name: Run cargo-tarpaulin
+        uses: actions-rs/tarpaulin@v0.1
+        with:
+          version: '0.12.4'
+          args: '-- --test-threads 1'
+      - name: Upload to codecov.io
+        uses: codecov/codecov-action@v1.0.2
+        with:
+          token:
```

Comment:
We don't need to specify a token, since this is a public repository. This can be removed :)

Reply:
I removed this but it seems to still fail? Perhaps it is because I'm trying from my fork of the repo.

Reply:
I think this is because we still don't have the project in CodeCov. Public repos shouldn't need a token.